extensions/ql-vscode/test/vscode-tests/activated-extension/databases/db-panel.test.ts

```diff
@@ -128,15 +133,93 @@ describe("Db panel UI commands", () => {
     );
 
     // Check db config
-    const dbConfigFilePath = path.join(
-      storagePath,
-      DbConfigStore.databaseConfigFileName,
-    );
     const dbConfig: DbConfig = await readJson(dbConfigFilePath);
     expect(dbConfig.selected).toBeDefined();
     expect(dbConfig.selected).toEqual({
       kind: SelectedDbItemKind.VariantAnalysisSystemDefinedList,
       listName,
     });
   });
+
+  it("should import from code search", async () => {
+    // This name is important, since other tests depend on only one list being present during the second suite run.
+    // See comment above.
+    // ;(
+    const listName = "my-list-1";
+    const mockServer = new MockGitHubApiServer();
+    mockServer.startServer();
+
+    await mockServer.loadScenario("code-search-success");
+
+    jest.spyOn(authentication, "getSession").mockResolvedValue({
+      id: "test",
+      accessToken: "test-token",
+      scopes: [],
+      account: {
+        id: "test",
+        label: "test",
+      },
+    });
+
+    const dbConfigPreparation: DbConfig = createDbConfig({
+      remoteLists: [
+        {
+          name: listName,
+          repositories: [],
+        },
+      ],
+    });
+
+    await writeJson(dbConfigFilePath, dbConfigPreparation);
+    await sleep(5000);
```

Comment:
I don't think we should have tests that have `sleep` because they're slow and/or flaky (because they rely on timings). This might be a personal opinion but I'd rather not have a test than have one that is slow/flaky. Happy to hear from others too.